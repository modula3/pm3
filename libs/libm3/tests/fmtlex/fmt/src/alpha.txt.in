# Last modified on Wed Mar 23 17:58:57 PST 1994 by heydon

# This file contains numbers that crashed "Float.ToDecimal" on the Alpha, but
# not on the DECstation.

l A -1 3.2767999999761581
