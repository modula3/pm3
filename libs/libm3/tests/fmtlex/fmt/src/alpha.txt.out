l A -1 3.2767999999761581
-> 16_75711bebe2360a40
-> "3.276799999976158e0" (decimal approximation)
-> "3.276799999976158"
-> "3.276799999976158d0"

